app/assets/stylesheets/theme/custom.css.scss

```diff
@@ -1,17 +1,15 @@
-select {
-  &#search_category {
-    background: url(asset-path("theme/dropdown_arrow.png")) no-repeat 95% center transparent;
-    background-size: 10px 5px;
-    border: none;
-    box-shadow: none;
-    font-weight: 400;
-    height: auto;
-    padding: 10px 22px 10px 4px;
-    width: 160px;
-    -webkit-appearance: none;
-    -moz-appearance: none;
-    appearance: none;
-  }
+select.form-control.search_options {
+  background: url(asset-path("theme/dropdown_arrow.png")) no-repeat 95% center transparent;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered -moz-appearance, -webkit-appearance, appearance, background, background-size, border, box-shadow, font-weight, height, padding, width